gms/impl/src/main/resources/mlModelESSearchQueryTemplate.json

```diff
@@ -0,0 +1,67 @@
+{
+  "function_score": {
+    "query": {
+      "bool": {
+        "should": [
+          {
+            "query_string": {
+              "query": "$INPUT",
+              "analyzer": "whitespace_lowercase",
+              "fields": [
+                "evaluationDatasets^0.065",
+                "trainingDatasets^0.075",
+                "description^0.035",
+                "platform^0.055",
+                "type^0.01"
+              ],
+              "default_operator": "OR"
+            }
+          },
+          {
+            "query_string": {
+              "query": "$INPUT",
+              "analyzer": "whitespace_lowercase",
+              "boost": 0.125,
```

Comment:
not sure if you intend to use this, you are already setting the boost values for individual fields?